std/format/internal/write.d

```diff
@@ -3031,9 +3031,8 @@ if (is(T == enum))
         auto w2 = appender!string();
 
         // val is not a member of T, output cast(T) rawValue instead.
-        put(w2, "cast(");
-        put(w2, T.stringof);
-        put(w2, ")");
+        static immutable prefix = "cast(" ~ T.stringof ~ ")";
```

Comment:
Ok with me to use enum here instead or inline the reference to `prefix`.

Reply:
enum is good.

Reply:
Fixed.